src/error-table.rkt

```diff
@@ -1,70 +1,157 @@
 #lang racket
 
-(require racket/set math/bigfloat)
+(require racket/set math/bigfloat racket/hash)
 (require "points.rkt" "syntax/types.rkt" "core/localize.rkt" "common.rkt"
-         "ground-truth.rkt" "syntax/sugar.rkt" "timeline.rkt" "programs.rkt")
+         "ground-truth.rkt" "syntax/sugar.rkt" "alternative.rkt" "programs.rkt"
+         "float.rkt" "config.rkt")
 
-(provide actual-errors predicted-errors)
+(provide actual-errors predicted-errors make-flow-table calculate-confusion
+         calculate-confusion-maybe)
```

Comment:
Why export these? Do we ever call anything except `predicted-errors` outside this file?

Reply:
I think this is an artifact from before I refactored all the table creation out of mainloop, I'll remove all the extra functions.